crates/swc/tests/tsc-references/readonlyRestParameters.2.minified.js

```diff
@@ -1,19 +1,2 @@
 //// [readonlyRestParameters.ts]
 import _to_consumable_array from "@swc/helpers/src/_to_consumable_array.mjs";
-function f0(a, b) {
-    f0(a, b), f1(a, b), f2(a, b);
-}
-function f1() {
-    for(var _len = arguments.length, args = Array(_len), _key = 0; _key < _len; _key++)args[_key] = arguments[_key];
-    f0.apply(void 0, _to_consumable_array(args)), f1("abc", "def"), f1.apply(void 0, [
-        "abc"
-    ].concat(_to_consumable_array(args))), f1.apply(void 0, _to_consumable_array(args));
-}
-function f2() {
-    for(var _len = arguments.length, args = Array(_len), _key = 0; _key < _len; _key++)args[_key] = arguments[_key];
-    f0.apply(void 0, _to_consumable_array(args)), f1("abc", "def"), f1.apply(void 0, [
-        "abc"
-    ].concat(_to_consumable_array(args))), f1.apply(void 0, _to_consumable_array(args)), f2("abc", "def"), f2.apply(void 0, [
-        "abc"
-    ].concat(_to_consumable_array(args))), f2.apply(void 0, _to_consumable_array(args));
-}
```

Comment:
We shouln't rely on inling to remove multi recursive functions, this needs to be fixed in future.

Reply:
#5709

Reply:
Oh.. I'll check if a graph is really required once I merge this PR and https://github.com/swc-project/swc/pull/5699

Reply:
It's needed. Just not in this case.